### PR TITLE
Add helper scripts to use `aflasim-sdk` to build plugins using conda env

### PR DESCRIPTION
**Pull request by prusse-martin**

### CODE REVIEW

**Review by BeneBr — APPROVED** [review]

**Review by dettmerramon — APPROVED** [review]

**Comment by ggrbill** [comment]
I would like to say that a description for this PR would clarify why you had to add this feature. 
Perhaps, add a description to register in this PR the reason why you are doing this.

File: src/alfasim_sdk/testing/fixtures.py [hunk 1]
```diff
@@ -120,17 +120,19 @@ def run(self, *, number_of_threads: int = -1) -> Results:
...
-        self._run_simulator(
-            self.get_runner()
-            + [
-                "--run-simulation",
-                f"--alfacase-file={str(self.alfacase_file)}",
-                f"--number-of-threads={number_of_threads}",
-            ],
-            cwd=project_folder,
-        )
+        try:
```
Comment (ggrbill) [inline src/alfasim_sdk/testing/fixtures.py:123]: Doubt:
I know that a generic error would be raised. But what is the name of this error? Which one would be raised?

File: src/alfasim_sdk/testing/fixtures.py [hunk 2]
```diff
@@ -120,17 +120,19 @@ def run(self, *, number_of_threads: int = -1) -> Results:
...
+            self._run_simulator(
```
Comment (ggrbill) [inline src/alfasim_sdk/testing/fixtures.py:124]: You changed this function to return an error code, shouldn't this error code be checked here?

**Review by ggrbill — APPROVED** [review]
Please, take a look at my questions.

File: src/alfasim_sdk/testing/fixtures.py (hunk 1 above)
Comment (prusse-martin) [reply src/alfasim_sdk/testing/fixtures.py:123]: `subprocess.CalledProcessError`

File: src/alfasim_sdk/testing/fixtures.py (hunk 2 above)
Comment (prusse-martin) [reply src/alfasim_sdk/testing/fixtures.py:124]: I will not return the error code, just let subprocess do the checking.

File: tests/results/test_result_reader.py [hunk 3]
```diff
@@ -68,7 +73,7 @@ def test_profiles(results: Results) -> None:
 
 
 def test_global_trends(results: Results) -> None:
-    assert results.list_global_trends() == ["timestep"]
+    assert list(map(str, results.list_global_trends())) == ["timestep"]
```
Comment (ggrbill) [inline tests/results/test_result_reader.py:76]: 👍🏼

**Pull request merged**